include/access_management.js

```diff
@@ -147,7 +147,8 @@ module.exports = function(pb) {
                 'ACCESS_ADMINISTRATOR': ls.get('ACCESS_ADMINISTRATOR'),
             };
         }
-        return null;
+        // Return an empty object instead of null, so things won't break after return
+        return {};
```

Comment:
The code wasn't handling a null return, it was going straight to acting upon an object.